trademapper/js/trademapper.yearslider.js

```diff
@@ -140,12 +145,13 @@ function($, d3) {
 					this.intervalId = null;
 					this.togglePlayButton(false);
 				}
-				// disable the slider
```

Comment:
Here you remove the comment, but not the next line that adds the class "disabled". Is that as it should be?

Reply:
I removed it because this disables the whole section, not just the slider.

Reply:
Ok, makes sense :)